pkg/api/short_url.go

```diff
@@ -212,13 +210,7 @@ func (sk8s *shortURLK8sHandler) createKubernetesShortURLsHandler(c *contextmodel
 
 	c.Logger.Debug("Creating short URL", "path", cmd.Path)
 	obj := shorturl.LegacyCreateCommandToUnstructured(cmd)
-
-	uid, err := shortid.Generate()
-	if err != nil {
-		c.JsonApiErr(http.StatusInternalServerError, "failed to generate uid", err)
-		return
-	}
-	obj.SetGenerateName(uid)
+	obj.SetGenerateName("u") // becomes a prefix
```

Comment:
`LegacyCreateCommandToUnstructured` sets `metadata.name`. is this needed?

Reply:
it would be when there is no legacy backing (that currently ignores generateName anyway) but it seems better to keep it so it works regardless of which storage backend is configured

Reply:
ah, i see what you mean now 👍